src/objective-c/tests/InteropTests.m

```diff
@@ -457,4 +457,99 @@ - (void)testRPCAfterClosingOpenConnections {
   [self waitForExpectationsWithTimeout:TEST_TIMEOUT handler:nil];
 }
 
+- (void)testAlternateDispatchQueue {
```

Comment:
Please make a separate test out of this, as it has nothing to do with interoperability.